app/serializers/subject_serializer.rb

```diff
@@ -3,7 +3,7 @@ class SubjectSerializer
   include FilterHasMany
 
   attributes :id, :metadata, :locations, :zooniverse_id,
-    :created_at, :updated_at, :href
+    :created_at, :updated_at, :href, :favorite
```

Comment:
This should be an optional attribute and turned on / off via the existing context setup and feature flag for the extra selection context.